homeassistant/components/systemmonitor/sensor.py

```diff
@@ -389,6 +389,7 @@ async def async_setup_entry(
     entities = []
     sensor_registry: dict[tuple[str, str], SensorData] = {}
     legacy_resources: list[str] = entry.options.get("resources", [])
+    loaded_entities: list[str] = []
```

Comment:
```suggestion
    loaded_resources: list[str] = []
```

Reply:
Tested with the two PR's together so this should be ok to go now